bot/cogs/custom_ui/page_turn_embed.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+import discord
+from discord.ext import commands
+
+
+class PageTurnView(discord.ui.View):
+    """discord.ui.View subclass for page-turn functionality between embeds
```

Comment:
Suggestion
Personally, I am a fan of numpy style doc strings. I think they play nicer with VSCode